coalib/output/ConsoleInteraction.py

```diff
@@ -54,7 +54,7 @@ def highlight_text(text, lexer=TextLexer(), style=None):
 
 
 STR_GET_VAL_FOR_SETTING = ("Please enter a value for the setting \"{}\" ({}) "
-                           "needed by {}: ")
+                           "needed by {} ")
```

Comment:
removing the colon? Why? The colon makes sense. Don't do what I say unless you have reason to believe it's right :P